server/app.js

```diff
@@ -376,6 +376,31 @@ app.put('/api/user/rename', async (req, res)=>{
 	}
 });
 
+//Delete brews based on author
+app.delete('/api/user/delete', async (req, res)=>{
+	const { username } = req.body;
+	const ownAccount = req.account && (req.account.username == username);
+	
+	if(!username)
+		return res.status(400).json({ error: 'Username and newUsername are required.' });
+	if(!ownAccount)
+		return res.status(403).json({ error: 'Must be logged in to change your username' });
+
+	try {
+		const brews = await HomebrewModel.getByUser(username, true, ['authors']);
+		await brews.forEach((brew)=>{
+			request.delete(`/api/${brew.googleId ?? ''}${brew.editId}`).send().end((err, res)=>{
+				if (err) reportError(err);
```

Comment:
`reportError` is not defined. This line can be removed.

If you call `deleteBrew()` directly, any error will already be caught by your try/catch block and handled there.